test/server-test.spec.js

```diff
@@ -0,0 +1,46 @@
+const supertest = require('supertest')(require('../server/routes/routes'))
+const expect = require('chai').expect
+
+const product = {
+    name: 'Lettuce',
+    price: 2.00,
+    description: 'Very fresh green romaine lettuce!'
+}
+
+describe('Server', () => {
+    describe('Routes', () => {
+        describe('/products GET URI', () => {
+            it('should send a valid JSON response', () => {
+                return supertest
```

Comment:
Way too light. Knowing that it sent a `200` with a `Content-Type` header is what you're testing here. You want to test that the content of the response is correct.